test/test.js

```diff
@@ -40,6 +51,35 @@ describe('module', function () {
 		expect($webhook.validateSecret(false, request)).to.be.false;
 		expect($webhook.validateSecret({}, request)).to.be.false;
 		expect($webhook.validateSecret({ webhookId: 125 }, request)).to.be.false;
+		expect($webhook.validateSecret(multiConfigSettings.crazyKeyString, request)).to.be.true;
+	});
+
+	it('returns only valid configId from multi configs', function () {
+		expect($webhook.getValidatedMultiConfigId(multiConfigSettings, request)).to.equal('crazyKeyString');
+
+		const reqWithBadHeader = {
+			headers: {},
+			body: {
+				webhook: {
+					id: 123
+				}
+			}
+		};
+
+		expect($webhook.getValidatedMultiConfigId(multiConfigSettings, reqWithBadHeader)).to.equal(undefined);
+
+		const reqWithBadId = {
+			headers: {
+				'x-zengine-webhook-key': 456
+			},
+			body: {
+				webhook: {
+					id: 987
+				}
+			}
+		};
+
+		expect($webhook.getValidatedMultiConfigId(multiConfigSettings, reqWithBadId)).to.equal(undefined);
```

Comment:
@tehpsalmist im almost certain there's a "better" version which is`expect(foo).to.be.undefined`

Reply:
Well what do ya know.